db.json

```diff
@@ -18416,4 +18476,4 @@
         "type": "pcdsdevices.happi.containers.GateValve",
         "z": 870.97
     }
-}
+}
```

Comment:
Yeah, I notice this like every other time and looked into fixing it but never proceeded since it's such a small thing. :/
We'd just have to add a `f.write('\n')` after our `json.dump` at https://github.com/pcdshub/happi/blob/a1b8cebc859160c6dccd0638c073d1ec241e37df/happi/backends/json_db.py#L143